app/scripts/views/settings/devices.js

```diff
@@ -42,9 +69,15 @@ define(function (require, exports, module) {
       devices.on('remove', this._onDeviceRemoved.bind(this));
     },
 
+    _formatDevicesList: function (devices) {
+      return _.each(devices, function (device) {
+        device.lastAccessTime = formatConnectedTimestamp(Number(device.lastAccessTime));
```

Comment:
yeap, sorry I had `map` in there before, will update. I think for now we shall keep it simple and if we have to reuse this we can modify this a bit in the future.